server/build/babel/preset.js

```diff
@@ -1,5 +1,15 @@
-const babelRuntimePath = require.resolve('babel-runtime/package')
-  .replace(/[\\/]package\.json$/, '')
+// Used to generate paths for hard-coding module aliases that don't include the
+// path back to the root of the filesystem. Important for when building and
+// running are done from different directories.
+const packageRelativeResolve = (path) => {
+  // We want paths relative to the first 'node_modules' we find in the path
+  const RELATIVE_START = 'node_modules/'
+
+  const absolutePath = require.resolve(path)
+    .replace(/[\\/]package\.json$/, '')
+
+  return absolutePath.substring(absolutePath.indexOf(RELATIVE_START) + RELATIVE_START.length)
+}
```

Comment:
You could, but you’d need to leave the `require.resolve` bits in the original file.